ios/HeliumWidget/HeliumBalanceWidget/HeliumBalanceWidgetEntryView.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  HeliumBalanceWidgetEntryView.swift
+//  Helium
+//
+//  Created by Luis F. Perrone on 1/18/22.
+//
+
+import WidgetKit
+import SwiftUI
+import Intents
+
+struct HeliumBalanceWidgetEntryView : View {
+  var entry: Provider.Entry
+    
+  var body: some View {
+      VStack(alignment: .leading) {
+        Spacer()
+        HStack(spacing: 0) {
+          Image("heliumLogo").resizable()
+            .frame(width: 32.0, height: 32.0)
+            .foregroundColor(Color(red: 71/255, green: 77/255, blue: 255/255))
+          Spacer()
+          VStack(alignment: .trailing, spacing: 4) {
+            Text("Helium (HNT)")
+              .bold()
+              .font(.system(size: 10.0))
+            Text("$\(String(format: "%.2f",entry.hntPrice))")
+              .font(.system(size: 12.0))
+          }
+
+        }
+        Divider()
+        Spacer()
+        HStack(alignment: .center) {
+          Text("\(String(format: "%.2f",entry.hntDailyEarnings)) HNT")
+            .bold()
+            .font(.system(size: 12.0))
+          Spacer()
+          Text("Jan 17")
```

Comment:
This hardcoded date will need to be updated. Could possibly add it to the SharedGroupPreferences

Reply:
Oops yeah that should actually just be set to a new date that is formatted to UTC. Let me fix that

Reply:
I believe it should be set to yesterday's date based on current UTC.

Reply:
After thinking about it some more, the widget should use the date from the app. If the app hasn't been foregrounded in more than 24 hours, the widget balance will be stale and we wouldn't want to show a current date.

Reply:
Sounds good I can pull in the date from the app and when it refreshes will pull from the widget API